components/patient-data/impl/src/main/java/org/phenotips/data/internal/PhenoTipsPatient.java

```diff
@@ -507,6 +507,7 @@ private void updateDisordersFromJSON(XWikiDocument doc, BaseObject data, XWikiCo
                 // as in constructor: make unmofidiable
                 this.disorders = Collections.unmodifiableSet(this.disorders);
 
+                data.set(DISORDER_PROPERTIES_OMIMID, null, context);
```

Comment:
why do we have two data.set(DISORDER_PROPERTIES_OMIMID, ...) on two neighbouring lines?